notifications/NotificationHelper.cpp

```diff
@@ -447,6 +447,53 @@ bool CNotificationHelper::CheckAndHandleNotification(const uint64_t DevRowIdx, c
 					return false;
 			}
 			break;
+		case pTypeLocation:
+			nexpected = 3;
+			if (nsize >= nexpected) {
+				// Here we should do something to calculate something around the Location like distance
+				double fDomLat;
+				double fDomLon;
+				int nValue;
+				std::string sValue;
+				std::string Latitude;
+				std::string Longitude;
+
+				uint64_t nDistance = 0;	// Just for testing, set the distance at 1000 meters
+				double fLat = atof(strarray[0].c_str());
+				double fLon = atof(strarray[1].c_str());
+				double fAlt = atof(strarray[2].c_str());
+
+				// Let's get the 'home' Location of this Domoticz instance from the Preferences
+				if (!m_sql.GetPreferencesVar("Location", nValue, sValue))
```

Comment:
@gizmocuz , is there an easier/better way of getting the Location data from Domoticz setting instead of going to the database each time? Aren't these variables in memory somewhere?

Reply:
@gizmocuz , any suggestions for improvement I could try to implemented?